hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/dispatcher/WritableStoreFactoryTest.java

```diff
@@ -63,7 +61,6 @@ private static Stream<Arguments> storeParameters() {
                 arguments(TokenService.NAME, WritableNftStore.class),
                 arguments(TokenService.NAME, WritableTokenStore.class),
                 arguments(TokenService.NAME, WritableTokenRelationStore.class),
-                arguments(FreezeService.NAME, WritableUpdateFileStore.class),
```

Comment:
Does something else need to be added here?

Reply:
I put back in this test case